services/pull/merge.go

```diff
@@ -668,7 +668,7 @@ func MergedManually(ctx context.Context, pr *issues_model.PullRequest, doer *use
 		return err
 	}
 
-	notify_service.MergePullRequest(baseGitRepo.Ctx, doer, pr)
```

Comment:
Wait, doesn't this introduce a regression?
Didn't we explicitly change it to this specific context to have a deadline that is different from the request deadline?
With the previous approach, it is merged in the background.
With the new approach, it cannot take longer than the request…